lnbits/core/templates/core/index.html

```diff
@@ -160,15 +160,19 @@
                     class="text-center text-grey-6"
                     v-if="authAction !== 'reset'"
                   >
-                    <p v-if="authAction === 'login'" class="q-mb-none">
+                    <p
+                      v-if="authAction === 'login' && {{ LNBITS_NEW_ACCOUNTS_ALLOWED }}"
```

Comment:
wouldn't this be true when `LNBITS_NEW_ACCOUNTS_ALLOWED` is `"False"`?

on the same file this is used:
 :allowed_new_users="{{ LNBITS_NEW_ACCOUNTS_ALLOWED | tojson }}"

Reply:
good one